CopilotKit/packages/react-ui/src/components/chat/Messages.tsx

```
@@ -1,223 +1,82 @@
+              UserMessage={UserMessage}
+              ImageRenderer={ImageRenderer}
+              onRegenerate={onRegenerate}
+              onCopy={onCopy}
+              onThumbsUp={onThumbsUp}
+              onThumbsDown={onThumbsDown}
+              markdownTagRenderers={markdownTagRenderers}
+            />
+          );
+          {
+            interrupt;
           }
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)